here-naksha-app-service/src/test/java/com/here/naksha/app/service/ReadFeaturesByBBoxHttpStorageTest.java

```diff
@@ -228,4 +235,18 @@ void tc0711_testGetByBBoxOnViewSpace() throws Exception {
     verify(1, getRequestedFor(endpointPath));
   }
 
+  @ParameterizedTest
+  @MethodSource("com.here.naksha.app.service.PropSearchTestUtil#queryParams")
```

Comment:
nice, I did not know `@MethodSource` could reference source from different class